articles/quickstart/native/android/06-calling-apis.md

```diff
@@ -86,6 +119,6 @@ client.newCall(request).enqueue(new Callback() {
 });
 ```
 
-Check if the request was made and if the response that came back was what you expected. 
+Check if the request was made and if the response that came back was what you expected.
```

Comment:
Is this information needed? the code is commented.

Reply:
not really. n1